docs/containers-signature.5.md

```diff
@@ -112,7 +112,7 @@ and MUST reject unexpected data in the `critical` object, or in the top-level ob
 ```json
 {
     "critical": {
-        "type": "atomic container signature",
+        "type": "container signature",
```

Comment:
This would immediately break all existing signatures, and is completely impractical.

Introducing a new value, and figuring out some migration plan (still breaking old consumers), is IMHO not at all worth it.

Reply:
Ok removed this change.